library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt

```diff
@@ -48,7 +58,7 @@ class DonutProgressView @JvmOverloads constructor(
      * Eg. when one line has 50% of total graph length,
      * setting this to 0.5f will result in that line being animated to 25% of total graph length.
      */
-    var masterProgress: Float = 1f
+    var masterProgress: Float = DEFAULT_MASTER_PROGRESS
         set(value) {
             if (value !in (0f..1f)) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/828b548c8186e32727ef08c33dcf88e758e87083/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L63"></span>Parentheses in (0f..1f) are unnecessary and can be replaced with: 0f..1f</td>
    </tr>
  </tbody>
</table>